HOWTOMOVE.md

```diff
@@ -8,6 +8,7 @@ It has 2 steps to move a pkg from other Kubernetes repos to `k8s.io/utils` repo:
 
 Copying should preserve all the git history associated with it.
 [Here](http://gbayer.com/development/moving-files-from-one-git-repository-to-another-preserving-history/) is a working approach.
+Note: You may need to use `--allow-unrelated-histories` if you get error when running `git pull` following the post above.
```

Comment:
Have you used that so far? I'd really like not to use that as it can be really confusing ...

Reply:
I used it twice: 1 month ago and yesterday.